### PR TITLE
Build navigation from data

### PR DESCRIPTION
**Pull request by RonaldDijkstra**

### CODE REVIEW

File: source/partials/header/_nav-primary.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<li class="nav-section primary">
+  <% primary_nav_items.each do |primary_nav_item| %>
+    <%= nav_link_to "#{primary_nav_item.title}", "#{primary_nav_item.path}", class: "root-link item-#{primary_nav_item.item} has-dropdown", "data-dropdown": "#{primary_nav_item.item}" %>
```
Comment (fatboypunk) [inline source/partials/header/_nav-primary.erb]: title path and item are all strings aren't they?

File: source/localizable/header/_nav-primary.erb [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
+<%
+all_nav_items = data["navigation_" + I18n.locale.to_s]
+nav_items = all_nav_items.clone
+%>
 
-  <% if root_locale == :de %>
-    <%= nav_link_to t("header.solutions"), "/oplossingen.html", class: "root-link item-solutions has-dropdown", "data-dropdown": "solutions" %>
+<li class="nav-section primary">
+  <% nav_items.primary.each do |primary_nav_item| %>
+    <%= nav_link_to "#{primary_nav_item.title}", "#{primary_nav_item.path}", class: "root-link item-#{primary_nav_item.class}", "data-dropdown": "#{primary_nav_item.class}" %>
```
Comment (fatboypunk) [inline source/localizable/header/_nav-primary.erb]: Same here, don't think you need the interpolation

Comment (RonaldDijkstra) [reply source/localizable/header/_nav-primary.erb]: Oh thanks

Comment (RonaldDijkstra) [reply source/localizable/header/_nav-primary.erb]: True

**Pull request closed without merging**